v2/internal/controllers/crd_apimanagement_20230501preview_test.go

```diff
@@ -0,0 +1,497 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	apim "github.com/Azure/azure-service-operator/v2/api/apimanagement/v1api20230501preview"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_ApiManagement_20230501preview_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+	rg := tc.NewTestResourceGroup()
+
+	tc.CreateResourceAndWait(rg)
+
+	// The v2 SKU has a much quicker start up time.
```

Comment:
TBH I'd be OK with deleting the other (slower) test if you think that makes sense.

Reply:
Could probably do in a separate PR (or I can do it once this PR merges if you're busying doing other things)

Reply:
Let's tidy up in a separate PR and then it's explicit what and why we are doing it

Reply:
Added #3695